<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSelectMultiParamDlg</class>
 <widget class="QDialog" name="QSelectMultiParamDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1365</width>
    <height>973</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1175</x>
     <y>936</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>3</y>
     <width>1150</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="m_C3IRadio">
        <property name="text">
         <string>C3I系统                          </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_DFZXRadio">
        <property name="text">
         <string>东风中心                                </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_KELRadio">
        <property name="text">
         <string>库尔勒                             </string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="m_CustomRadio">
        <property name="text">
         <string>自定义范围                                </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>最小表号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_MinlineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>最大表号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_MaxlineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_SetCustom">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>351</y>
     <width>1001</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>参数表列表</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="m_ParamTableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1072</x>
     <y>350</y>
     <width>281</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>所选参数列表</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="m_SelectedtableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="m_FormTableWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>40</y>
     <width>1341</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_UpButton">
   <property name="geometry">
    <rect>
     <x>1018</x>
     <y>705</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>向上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_DownButton">
   <property name="geometry">
    <rect>
     <x>1018</x>
     <y>741</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>向下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_BottomButton">
   <property name="geometry">
    <rect>
     <x>1018</x>
     <y>669</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>置底</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_TopButton">
   <property name="geometry">
    <rect>
     <x>1018</x>
     <y>633</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>置顶</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_AddButton">
   <property name="geometry">
    <rect>
     <x>1018</x>
     <y>561</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_DelButton">
   <property name="geometry">
    <rect>
     <x>1018</x>
     <y>597</y>
     <width>49</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>m_FormTableWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>buttonBox</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>m_KELRadio</zorder>
  <zorder>m_DFZXRadio</zorder>
  <zorder>m_C3IRadio</zorder>
  <zorder>label</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QSelectMultiParamDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QSelectMultiParamDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
